### DISCUSSION TITLE
Up-to-date Fedora 37 RPM package

### DISCUSSION THREAD

**Discussion by muety** [rustdesk/rustdesk#5009] [Feature Request]
Currently, the RPM package is for Fedora 28, while latest Fedora version is 37. Would be cool if somebody could look into updating this! :pray:

**Comment by muety** [comment]
Never mind, I figured the latest 1.2.1 release has a working RPM for Fedora 37, thanks!

**Reply by andy-page-developer** [reply]
@muety  Would be so kind as to shared where you got this. I'm still on 1.1.9 and can't find where to update/upgrade. I'm on Fedora 38

**Reply by muety** [reply]
I downloaded the RPM from the latest [release](https://github.com/rustdesk/rustdesk/releases/tag/1.2.1).
